bundles/AdminBundle/src/Controller/Admin/Asset/AssetController.php

```diff
@@ -426,7 +428,7 @@ public function existsAction(Request $request): JsonResponse
      * @param Request $request
      * @param Config $config
      *
-     * @return array{success: bool, asset: ?Asset}
```

Comment:
Why did you remove this? It's still true, right? 
The changes in this file look to me like a conflict resolution gone wrong.
@blankse can you have a look, please?